include/deal.II/base/thread_management.h

```diff
@@ -692,16 +692,18 @@ namespace Threads
     private:
       RT value;
     public:
+      typedef RT &reference_type;
+
       inline return_value () : value() {}
 
-      inline RT get () const
+      inline reference_type get ()
```

Comment:
Is there anything that prevents this function from being `const`?

Reply:
Yes. If you make it `const`, the return type also has to be a `const` reference and you can't `std::move` the object that is returned.

Reply:
Good point.